text/0000-symbol-name-mangling-v2.md

```diff
@@ -330,48 +543,56 @@ std::iter::Chain<
   std::iter::Once<(u32, usize)>>
 ```
 
-It would make for a long symbol name if this types is used (maybe repeatedly) as a generic argument somewhere. C++ has the same problem with its templates; which is why the Itanium mangling introduces the concept of compression. If a component of a definition occurs more than once, it will not be repeated and instead be emitted as a substitution marker that allows to reconstruct which component it refers to. The scheme proposed here will use the same approach.
+It would make for a long symbol name if this type is used (maybe
+repeatedly) as a generic argument somewhere. C++ has the same problem
+with its templates; which is why the Itanium mangling introduces the
+concept of compression. If a component of a definition occurs more than
+once, it will not be repeated and instead be emitted as a substitution
+marker that allows to reconstruct which component it refers to. The
```

Comment:
Is "substitution" still applicable, or should we go with "backreference" everywhere?

Reply:
I think it's fine to leave it as is. Back references are still used "substituted for" so taking about substitutions isn't wrong. And the wording of the paragraph would get more complicated for what I don't perceive as an unequivocal improvement.